### ISSUE TITLE
Incorrect tooltip text when terminal is maximized

### ISSUE THREAD

**Issue by encladeus** [microsoft/terminal#5693]
# Environment

```none
Windows build number: [10.0.18363.778]
Windows Terminal version (if applicable): Current master 44689b93

Any other software? No
```

# Steps to reproduce

Open terminal and maximize the window.
Hover the mouse over the *restore* button.

# Expected behavior
Displays the tooltip *Restore Down*

# Actual behavior

Displays the tooltip *Maximize*

**Comment by DHowett-MSFT** [comment]
Yup.

**Comment by DHowett-MSFT** [comment]
We knew this was a problem when we first added automation text (for accessibility).

**Comment by Chips1234** [comment]
Yes, it should be "restore down". I couldn't figure out how to do that.

**Comment by Chips1234** [comment]
Is there a possibility that we can use the system default buttons? They come with tooltips.

**Comment by DHowett** [comment]
There is no possibility of us using the system buttons. :(
